docs/start/install.mdx

```diff
@@ -197,7 +197,8 @@ Additionally, Qiskit only supports the CPython implementation of the Python lang
 
   - Linux x86_64 (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification).
   - macOS x86_64 (10.12 or later)
-  - Windows 64 bit
+  - macOS ARM64 (10.15 or newer)
```

Comment:
I just checked the wheels and the arm64 wheels are only compatible with macos 11.0 or newer.
```suggestion
  - macOS ARM64 (11.0 or newer)
```